extensions/amp-form/0.1/amp-form.js

```diff
@@ -218,10 +219,12 @@ export class AmpForm {
         requireAmpResponseSourceOrigin: true,
       }).then(response => {
         this.actions_.trigger(this.form_, 'submit-success', null);
+        this.analyticsEvent_('amp-form-submit-success');
```

Comment:
These need to be documented. Please see how access and slides document this and use the same convention?